.github/workflows/rust.yml

```diff
@@ -96,7 +96,30 @@ jobs:
       - name: Check clippy
         working-directory: ${{ matrix.directory }}
         if: always()
-        run: cargo +${{ matrix.toolchain }} make clippy -- -D warnings
+        run: |
+          cargo +${{ matrix.toolchain }} make clippy --message-format=json \
+            | clippy-sarif \
+            | jq '.runs[].results |= unique' \
+            | tee clippy.sarif \
+            | sarif-fmt
+
+          jq -e '.runs[].results == []' clippy.sarif> /dev/null
```

Comment:
```suggestion
          jq -e 'isempty(.runs[].results)' clippy.sarif > /dev/null
```

Reply:
To clarify the `-e` flag:

> Sets the exit status of jq to 0 if the last output values was neither false nor null, 1 if the last output value was either false or null, or 4 if no valid result was ever produced. Normally jq exits with 2 if there was any usage problem or system error, 3 if there was a jq program compile error, or 0 if the jq program ran.

* Exit Code 0: Success (if empty)
* Exit Code 1: Error (if `results` has at least one element)

Reply:
This is not working as expected, I reverted the change.

Reply:
Good call, `isempty` did not work as I expected :/